app/converter.py

```diff
@@ -18,7 +18,7 @@ def extract_cell_metdata(df_c_md):
     df_cell_md[LABEL.AH.value] = [df_c_md[LABEL.AH.value]]
     df_cell_md[LABEL.FORM_FACTOR.value] = [df_c_md[LABEL.FORM_FACTOR.value]]
     df_cell_md[LABEL.TEST.value] = [df_c_md[LABEL.TEST.value]]
-    df_cell_md[LABEL.MAPPING.value] = [df_c_md[LABEL.MAPPING.value]]
+    # df_cell_md[LABEL.MAPPING.value] = [df_c_md[LABEL.MAPPING.value]]
```

Comment:
Can we delete this line?

Reply:
mapping is an interesting one. We should decide if we're going to support this usage pattern